kuma/static/js/wiki.js

```diff
@@ -539,13 +539,29 @@
     });
 
     /*
-        Toggle kumascript error detail pane
+        Kumascript error detected
     */
-    $('.kserrors-details-toggle').toggleMessage({
-        toggleCallback: function() {
-            $('.kserrors-details').toggleClass('hidden');
-        }
-    });
+    // is there an error?
+    var $kserrors = $('#kserrors');
+    if($kserrors.length > 0){
+        // enable the details toggle
+        var $kserrorsToggle = $kserrors.find('.kserrors-details-toggle');
+        var $kserrorsDetails = $kserrors.find('.kserrors-details');
+        $kserrorsToggle.toggleMessage({
+            toggleCallback: function() {
+                $kserrorsDetails.toggleClass('hidden');
+            }
+        });
+        // loop through error list and log errors
+        var $kserrorsList = $kserrors.find('#kserrors-list');
```

Comment:
We can probably get by with a single top-level selector here. `.find()` is sort of like a descendant selector.

```javascript
var $kserrorsList = $('#kserrors-list');
```